pygmo/_py_algorithms.py

```diff
@@ -0,0 +1,562 @@
+# Copyright 2020 PaGMO development team
+#
+# This file is part of the pygmo library.
+#
+# This Source Code Form is subject to the terms of the Mozilla
+# Public License v. 2.0. If a copy of the MPL was not distributed
+# with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import random
```

Comment:
Could you please move these imports in the functions where they are used? I'd rather have as few 3rd party imports as possible in the global scope.

Reply:
I've moved most of the imports. The only one left is typing, since it is used to define the type hints in function signatures. That is not really necessary, but helpful for static type checking. Defined in https://www.python.org/dev/peps/pep-0483/

Reply:
No worries, those can stay :)